slack-api-model-kotlin-extension/src/main/kotlin/com/slack/api/model/kotlin_extension/block/composition/OptionGroupObjectBuilder.kt

```diff
@@ -27,8 +26,13 @@ class OptionGroupObjectBuilder private constructor(
         label = PlainTextObject(text, emoji)
     }
 
-    fun options(options: List<OptionObject>) {
-        this.optionContainer.underlying.addAll(options)
+    /**
+     * An array of option objects that belong to this specific group. Maximum of 100 items.
+     *
+     * @see <a href="https://api.slack.com/reference/block-kit/composition-objects#option_group">Option group object documentation</a>
+     */
+    fun options(builder: OptionObjectDsl.() -> Unit) {
```

Comment:
As this method is not `option` but `options`, changing it this way is irrelevant. `OptionObjectDsl` provides only a way to construct a single `OptionObject`.

Reply:
I think this conversation got lost after the rebase. Is it still applicable? I was doing this to make `options()` behave the way `elements()` does on some of the block builders. For example:

```kotlin
// ...
optionGroup {
  label("Some label")
  options {
    option {
      // Option data here
    }
    option {
      // Option data here
    }
  }
}
```

Otherwise, the builder methods from OptionObjectDsl will be unavailable and the user would have to construct each option object by hand

Reply:
Edited my above comment for clarity.

Reply:
Thanks for clarifying this! I was a bit confused when I left this comment. I just wrote some code with your latest revision and I understand how it works. LGTM.